dashboard/lib/school_info_deduplicator.rb

```diff
@@ -0,0 +1,45 @@
+module SchoolInfoDeduplicator
```

Comment:
It seems this is only used in models (`User` and `Pd::Enrollment`). Instead of lib, it could be implemented as a [concern](https://github.com/code-dot-org/code-dot-org/tree/c67c4061e5ba0d13f087fe45e56ffed889e877f1/dashboard/app/models/concerns).